docs/plans/regression-view-association-plan.md

```
@@ -0,0 +1,182 @@
+1. Plain `test_details` — preferred (used by regressed tests in component reports)
+2. `test_details:<viewName>` — view-specific composite key (used by regression objects)
+3. First `test_details:*` — fallback to any composite key
+
+`generateTestDetailsReportLink` updated to accept an optional `viewName` parameter.
+
+### 9.2 Per-file changes
+
+| File | Object type | Strategy |
+|------|------------|----------|
+| `RegressionRedirect.js` | Regression | Prefers `-main` view key, falls back to `getTestDetailsLink` |
+| `ComponentReadinessIndicator.js` | Regression | Uses `getTestDetailsLink(links, `${release}-main`)` |
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)